Makefile

```diff
@@ -2,6 +2,9 @@
 test:
 	nvim --headless --noplugin -u scripts/minimal.vim -c "PlenaryBustedDirectory tests/plenary/ {minimal_init = 'tests/minimal_init.vim', sequential = true}"
 
+test1:
+	nvim --headless --noplugin -u scripts/minimal.vim -c "PlenaryBustedDirectory tests/plenary/$(TEST1) {minimal_init = 'tests/minimal_init.vim', sequential = true}"
+
```

Comment:
If you want to add this to the `Makefile` targets I think it should be it's separated PR. But if @tjdevries or @Conni2461 thinks it's okay to leave it in this one, then it won't be needed.

Reply:
Removed the change. I'll maintain it locally.